pwiz_tools/Skyline/TestTutorial/TargetedMSMSTutorialTest.cs

```diff
@@ -1360,5 +1361,47 @@ private static void ValidateScoresMatch(double expectedScore, PropertyGrid pg, V
             Assert.AreEqual(expectedScore.ToString(CultureInfo.CurrentCulture), score?.ToString());
             Assert.AreEqual(expectedScore, dlg.GraphItem.SpectrumInfo.Score);
         }
+
+        private void ValidatePeakTooltips()
+        {
+            // 191, 95; 169, 175
+            var testData = new Dictionary<Point, string>()
+            {
+                {new Point(191, 95), "Observed m/z:\t951.6229\nIntensity:\t3814516\nRank:\t1\nMatched Ions\tIon m/z (calculated)\ny8\t951.4782  -152.1 ppm"},

```

Comment:
This test seems to fail in French, Chinese and Japanese.

Reply:
Thank you, Nick. I think it's the numbers formatting.